wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -1079,4 +1034,21 @@ private extension ZMConversation {
         }
     }
 
+    private var avsConversationTypeForOneOnOne: AVSConversationType {
+        guard
+            let context = managedObjectContext,
+            let featureConfig = FeatureRepository(context: context).fetchConferenceCalling().config,
+            featureConfig.useSFTForOneToOneCalls
+        else {
+            return .oneToOne
+        }
+
+        switch messageProtocol {
+        case .mls:
+            return .mlsConference
+        case .proteus, .mixed:
+            return .conference
+        }
+    }
```

Comment:
I feel like this should be a function `(ZMConversation) -> AvSConversationType`. Could we move this to the call center?